src/example-types/esTwoLevelAggregation.js

```diff
@@ -72,7 +75,17 @@ module.exports = {
                 key: bucket.key || key,
                 doc_count: bucket.doc_count,
               },
-              bucket.twoLevelAgg
+              bucket.twoLevelAgg ||
+                _.find(
+                  value =>
+                    !F.cascade(['value', 'values', 'value_count'], value) &&
```

Comment:
I don't think having `value_count` in here is right. The value_count agg has `.value` (https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-valuecount-aggregation.html). I don't think any metric agg has a `value_count` property on it